compiler/tests-common-jvm6/tests/org/jetbrains/kotlin/test/clientserver/TestProcessServer.kt

```diff
@@ -49,6 +49,8 @@ fun getBoxMethodOrNull(aClass: Class<*>): Method? {
 //Use only JDK 1.6 compatible api
 object TestProcessServer {
 
+    val DEBUG_TEST = "--debugTest"
```

Comment:
'private const val'

Reply:
I made it public to make it accessible from test class, maybe extract it somewhere?